### DISCUSSION TITLE
osu! android automatically crash when I play singleplayer

### DISCUSSION THREAD

**Discussion by NamOpt** [ppy/osu#25586] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

when i click the play button, is automatically crash

### Screenshots or videos

https://github.com/ppy/osu/assets/120299835/86230146-2fcf-45b3-b4ac-87fcd68ef522

### Version

2023.1114-1

### Logs

[database.log](https://github.com/ppy/osu/files/13462728/database.log)
[input.log](https://github.com/ppy/osu/files/13462729/input.log)
[network.log](https://github.com/ppy/osu/files/13462730/network.log)
[performance.log](https://github.com/ppy/osu/files/13462731/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13462732/runtime.log)

**Comment by bdach** [comment, marked as answer]
Judging from

```
2023-11-25 00:55:46 [verbose]: BAD_SURFACE failure in SwapBuffers suppressed
```

and

```
2023-11-25 00:36:45 [verbose]: GL Renderer:                Mali-G71
```

the most likely situation here is going to be "possibly broken driver, we can't support this device ever / until we get our hands on one where this happens and then debug it".
